tests/pytests/test_profile.py

```diff
@@ -231,13 +231,13 @@ def testProfileVector(env):
   env.assertEqual(env.cmd("FT.DEBUG", "VECSIM_INFO", "idx", "v")[-1], 'HYBRID_ADHOC_BF')
 
   for i in range(6, 10001):
-    conn.execute_command('hset', str(i), 'v', 'babababa', 't', "hello world")
+    conn.execute_command('hset', str(i), 'v', 'bababada', 't', "hello world")
```

Comment:
why the change of the query? scenario for 2 batches?

Reply:
Yes... To make these further from the query vector than `1`, but closer than `5`